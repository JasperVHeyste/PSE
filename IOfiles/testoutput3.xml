<?xml version="1.0" encoding="UTF-8"?>
<SYSTEM>
	<DEVICE>
		<name>Office_Printer1</name>
		<emissions>3</emissions>
		<speed>40</speed>
	</DEVICE>
	<DEVICE>
		<name>Office_Printer2</name>
		<emissions>2</emissions>
		<speed>30</speed>
	</DEVICE>
	<DEVICE>
		<name>Office_Printer3</name>
		<emissions>5</emissions>
		<speed>40</speed>
	</DEVICE>
	<DEVICE>
		<name>Office_Printer4</name>
		<emissions>100</emissions>
		<speed>40</speed>
	</DEVICE>
	<DEVICE>
		<name>Office_Printer5</name>
		<emissions>3</emissions>
		<speed>4000</speed>
	</DEVICE>
	<DEVICE>
		<name>Office_Printer6</name>
		<emissions>300</emissions>
		<speed>4000</speed>
	</DEVICE>
	<DEVICE>
		<name>Office_Printer7</name>
		<emissions>0</emissions>
		<speed>1</speed>
	</DEVICE>
	<JOB>
		<jobNumber>1</jobNumber>
		<pageCount>2</pageCount>
		<userName>SergeDemeyer</userName>
	</JOB>
	<JOB>
		<jobNumber>2</jobNumber>
		<pageCount>3</pageCount>
		<userName>anonymous_user</userName>
	</JOB>
	<JOB>
		<jobNumber>3</jobNumber>
		<pageCount>3</pageCount>
		<userName>anonymous_user</userName>
	</JOB>
	<JOB>
		<jobNumber>4</jobNumber>
		<pageCount>53</pageCount>
		<userName>Jasper</userName>
	</JOB>
	<JOB>
		<jobNumber>5</jobNumber>
		<pageCount>32</pageCount>
		<userName>Jens</userName>
	</JOB>
	<JOB>
		<jobNumber>6</jobNumber>
		<pageCount>6</pageCount>
		<userName>Kyrill</userName>
	</JOB>
	<JOB>
		<jobNumber>7</jobNumber>
		<pageCount>2</pageCount>
		<userName>SergeDemeyer</userName>
	</JOB>
	<JOB>
		<jobNumber>8</jobNumber>
		<pageCount>3</pageCount>
		<userName>anonymous_user</userName>
	</JOB>
	<JOB>
		<jobNumber>9</jobNumber>
		<pageCount>3</pageCount>
		<userName>anonymous_user</userName>
	</JOB>
	<JOB>
		<jobNumber>10</jobNumber>
		<pageCount>53</pageCount>
		<userName>unknown</userName>
	</JOB>
</SYSTEM>
