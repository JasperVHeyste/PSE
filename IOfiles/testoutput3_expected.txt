Printer 'Office_Printer1' finished job:
    Number: 1
    Submitted by 'SergeDemeyer'
    2 pages
Job 1 was made CO2 neutral with the support of Students 4 Climate.
Printer 'Office_Printer2' finished job:
    Number: 2
    Submitted by 'anonymous_user'
    3 pages
Job 2 was made CO2 neutral with the support of Students 6 Climate.
Printer 'Office_Printer3' finished job:
    Number: 3
    Submitted by 'anonymous_user'
    3 pages
Printer 'Office_Printer4' finished job:
    Number: 4
    Submitted by 'Jasper'
    53 pages
Printer 'Office_Printer5' finished job:
    Number: 5
    Submitted by 'Jens'
    32 pages
Job 5 was made CO2 neutral with the support of Students 5 Climate.
Printer 'Office_Printer6' finished job:
    Number: 6
    Submitted by 'Kyrill'
    6 pages
Job 6 was made CO2 neutral with the support of Students 6 Climate.
Printer 'Office_Printer7' finished job:
    Number: 7
    Submitted by 'SergeDemeyer'
    2 pages
Printer 'Office_Printer1' finished job:
    Number: 8
    Submitted by 'anonymous_user'
    3 pages
Printer 'Office_Printer2' finished job:
    Number: 9
    Submitted by 'anonymous_user'
    3 pages
Printer 'Office_Printer3' finished job:
    Number: 10
    Submitted by 'unknown'
    53 pages
Total CO2 emissions: 7503 gram.
